pandas/tests/io/json/test_pandas.py

```diff
@@ -607,19 +605,18 @@ def __str__(self) -> str:
             df_mixed.to_json()
 
         # default_handler should resolve exceptions for non-string types
-        assert df_nonprintable.to_json(
-            default_handler=str
-        ) == '{{"A":{{"0":"{hex}"}}}}'.format(hex=hexed)
-        assert df_mixed.to_json(
-            default_handler=str
-        ) == '{{"A":{{"0":"{hex}"}},"B":{{"0":1}}}}'.format(hex=hexed)
+        assert (
+            df_nonprintable.to_json(default_handler=str) == f'{{"A":{{"0":"{hexed}"}}}}'
```

Comment:
pls split into lines:

```
result = df_nonprintable...
expected = f'...'
assert result == expected
```

Reply:
Done. I also split the test below this one into 3 lines.